include/libcyphal/transport/defines.hpp

```diff
@@ -28,11 +30,71 @@ using TransferId = std::uint64_t;
 
 struct ProtocolParams final
 {
-    NodeId        max_nodes;
-    std::size_t   mtu_bytes;
-    std::uint64_t transfer_id_modulo;
+    TransferId  transfer_id_modulo;
+    std::size_t mtu_bytes;
+    NodeId      max_nodes;
+
+    ProtocolParams(TransferId _transfer_id_modulo, std::size_t _mtu_bytes, NodeId _max_nodes)
```

Comment:
Why do we need an explicit constructor here? Can we not use aggregate initialization? Same question applies to the other definitions.

Reply:
Probably we don't need them. It's my swift language habits ;) - we remove them all.